src/OSPSuite.InstallationValidator.Core/Domain/LogWatcher.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.IO;
+using OSPSuite.InstallationValidator.Core.Events;
+using OSPSuite.Utility;
+using OSPSuite.Utility.Events;
+
+namespace OSPSuite.InstallationValidator.Core.Domain
+{
+   public class LogWatcher : IDisposable
```

Comment:
Why no interface for this guy? That would make the testing maybe a bit easier maybe?